lldb/include/lldb/Utility/StructuredData.h

```diff
@@ -290,6 +290,9 @@ class StructuredData {
 
     void GetDescription(lldb_private::Stream &s) const override;
 
+    static ArraySP SplitString(llvm::StringRef s, char separator, int maxSplit,
+                               bool keepEmpty);
```

Comment:
Could you document this method ? It's not clear what `maxSplit` & `keepEmpty` do

Reply:
Moved. New document is in the header file, explaining all params and return.